bakery/env/prod.json

```diff
@@ -2,10 +2,12 @@
   "ENV_NAME": "production",
   "COPS_TARGET": "https://cops.openstax.org/api",
   "S3_PDF_BUCKET": "ce-pdf-spike",
-  "S3_DIST_BUCKET": "ce-contents-cops-distribution-373045849756",
-  "S3_QUEUE_STATE_BUCKET": "",
-  "QUEUE_FILENAME": "",
+  "S3_DIST_BUCKET": "web-hosting-content-gatekeeper",
+  "S3_QUEUE_STATE_BUCKET": "web-hosting-content-queue-state",
+  "QUEUE_FILENAME": "distribution-queue.json",
   "FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-books.json",
-  "PIPELINE_TICK_INTERVAL": "",
-  "MAX_BOOKS_PER_TICK": ""
+  "PIPELINE_TICK_INTERVAL": "24h",
+  "MAX_BOOKS_PER_TICK": "50",
+  "DIST_BUCKET_AKI_SECRET_NAME": "((prod-web-hosting-content-gatekeeper-access-key-id))",
+  "DIST_BUCKET_SAK_SECRET_NAME": "((prod-web-hosting-content-gatekeeper-secret-access-key))"
```

Comment:
Just to check my understanding: The idea is the same credentials get used for all three pipelines with this change, correct (though perhaps meant to be temporary since the name is specific to `dist`)? If so, will we need to extend the corresponding IAM user policy so that the production PDF pipeline (which will I believe now incorporate `((prod-web-hosting-content-gatekeeper-access-key-id))`) can write to its current s3 bucket (which I think is currently actually in our sandbox AWS, but I expect will move at some point)?

Reply:
That was not the idea. I think there's some changes to be made for how these env files are handled in the future, because right now, the variables used by pdf vs distribution/gdoc are exclusive to those pipelines and I don't believe there is any intermixing. Because of this, this secret name will not and should not be embedded within any PDF pipeline, only gdoc/dist, since they share the same target bucket.

Additionally staging/prod names are specific to pdf COPS, but we've repurposed them to mean sandbox/prod for the purposes of distribution/gdoc pipeline deployments, which to me is gonna end up being really confusing. Hopefully this is good enough for today, since it seems to be secure enough and we are already behind the deadline for moving content-distribution to production infrastructure. I'm open to ideas on how to restructure the env distinctions going forward. Maybe we should make a investigation/decomp card?

I think what you've identified is some additional infrastructure we will need to have the PDF pipelines also target production account s3 buckets (which will make use of a different IAM User and different credentials, I'm guessing, since the pdf pipeline will be accessing different resources than the other pipelines)

Reply:
Agreed on the general statements above as far as deferring thinking about options to restructure / cleanup in a separate card. Mostly for this one, I'm wondering whether once merged if it would break PDF production since `./build pipeline pdf prod` with the changes produces a modified s3 resource definition for me 🤷‍♂️ :

```
  - name: s3
    type: s3
    source:
      bucket: ce-pdf-spike
      access_key_id: ((prod-web-hosting-content-gatekeeper-access-key-id))
      secret_access_key: ((prod-web-hosting-content-gatekeeper-secret-access-key))
      skip_download: true
```

Reply:
yeah, that would break stuff. that should not be present in the pdf pipeline. I will correct that.

Reply:
good catch

Reply:
Added in keys specifically for pdf s3 target secret names (just used the sandbox keys for both for now, since both staging + prod COPS target a sandbox bucket - this might be the temporary piece you are referring to :smile: )